src/YDialogSpy.cc

```diff
@@ -514,7 +514,12 @@ void YDialogSpy::exec()
 
             continue;
         }
-        else if ( event->widget() == priv->upButton ) priv->moveSelectedUp();
+
+        // check for NULL which is generated for timeout events which can be
+        // triggered by the original dialog
+        if (!event->widget()) continue;
```

Comment:
Oh, sorry, the point is that when the detailed widget properties are not displayed the table widget (`priv->propTable`) is NULL. When the timeout event is received it's widget is NULL.

In that case this condition is true:
```
if ( event->widget() == priv->propTable )  priv->editProperty()
```
simply because `NULL == NULL` and then `priv->editProperty()` is called. And it wants to read the current value from the `propTable` widget (which is NULL)... :boom: